pagerduty/manifest.json

```diff
@@ -18,7 +18,7 @@
     "configuration": "README.md#Setup",
     "support": "README.md#Support",
     "changelog": "CHANGELOG.md",
-    "description": "Monitor Pagerduty services and interact with incidents directly from Datadog dashboard",
+    "description": "Monitor your PagerDuty incidents from your Datadog dashboard",
```

Comment:
Had to make this under 70 characters as it was failing a test due to length being too long. I think Datadog updated the description length limit to be shorter than it was before.